lib/visualize_packwerk/package_relationships.rb

```diff
@@ -6,25 +6,6 @@ class PackageRelationships
 
     OUTPUT_FILENAME = T.let('packwerk.png'.freeze, String)
 
-    sig { void }
-    def initialize
```

Comment:
Before, we tried to color each node based on team owners. This was extra unnecessary complexity and it forced us to raise if there are more than X unique colors.